internal/services/compute/orchestrated_virtual_machine_scale_set_resource_other_test.go

```diff
@@ -28,6 +28,32 @@ func TestAccOrchestratedVirtualMachineScaleSet_priority(t *testing.T) {
 	})
 }
 
+func TestAccOrchestratedVirtualMachineScaleSet_zoneUpdate(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_orchestrated_virtual_machine_scale_set", "test")
+	r := OrchestratedVirtualMachineScaleSetResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.singleZone(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		{
+			Config: r.multipleZone(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		{
+			Config: r.singleZone(data),
```

Comment:
My understanding is that we can add new zones but can't remove existing zones, so will this step recreate the resource? If so, we shouldn't have this step

Reply:
You're correct. It will recreate the resource. I have removed this step. Thanks.